doc/misc/npm-config.md

```diff
@@ -655,7 +655,7 @@ process is not aborted.
 * Type: Boolean
 
 Output parseable results from commands that write to
-standard output.
+standard output. For `npm search`, this will be tab-separated table format.
```

Comment:
Good start. This is yet more evidence that [`doc/misc/config.md` must be destroyed](https://github.com/npm/npm/issues/7185), because now I really want this to include what the parseable form means for other commands (which is totally out of scope for this PR – DON'T DO THIS).